Lib/hmac.py

```diff
@@ -81,15 +78,17 @@ def _init_old(self, key, msg, digestmod):
         self._inner = digest_cons()
         self.digest_size = self._inner.digest_size
 
+        import warnings
+
         if hasattr(self._inner, 'block_size'):
             blocksize = self._inner.block_size
             if blocksize < 16:
-                _warnings.warn('block_size of %d seems too small; using our '
+                warnings.warn('block_size of %d seems too small; using our '
```

Comment:
Can you align the ' of default with the ' of block_size please? same for 'Assuming